src/transport_portals4.c

```diff
@@ -271,27 +271,28 @@ shmem_transport_init(long eager_size)
                     &ni_limits,
                     &shmem_transport_portals4_ni_h);
     if (PTL_OK != ret) {
-        fprintf(stderr, "[%03d] ERROR: PtlNIInit failed: %d\n",
-                shmem_internal_my_pe, ret);
+        RETURN_ERROR_MSG("PtlNIInit failed: %d\n", ret);
         return ret;
     }
 
 #ifdef ENABLE_REMOTE_VIRTUAL_ADDRESSING
     if ((PTL_TARGET_BIND_INACCESSIBLE & ni_limits.features) == 0) {
-        fprintf(stderr,
-                "[%03d] ERROR: Remote virtual addressing feature enabled, but Portals\n"
-                "doesn't support PTL_TARGET_BIND_INACCESSIBLE.  Aborting.\n",
-                shmem_internal_my_pe);
+        RETURN_ERROR_MSG("Remote virtual addressing feature enabled, but Portals\n"
+                         RAISE_PE_PREFIX
+                         "doesn't support PTL_TARGET_BIND_INACCESSIBLE.  Aborting.\n",
+                         shmem_internal_my_pe);
```

Comment:
The ```RAISE_PE_PREFIX``` macro is a string that prints the PE id, so it is required.  This is a bit messy, but it seemed like the most reasonable way to allow multi-line strings in the _MSG macros.